Core/resources/roles.mustache

```diff
@@ -0,0 +1,39 @@
+  "roles": [
```

Comment:
This file shouldn't be in the delivered jar.  If you use it for testing put it into resources-test

Reply:
removed.